assets/global.js

```diff
@@ -529,32 +530,51 @@ class SliderComponent extends HTMLElement {
     this.update();
   }
 
+  resetPages() {
+    this.sliderItems = this.slider.querySelectorAll('li');
+    this.initPages();
+  }
+
   update() {
+    const previousPage = this.currentPage;
     this.currentPage = Math.round(this.slider.scrollLeft / this.sliderLastItem.clientWidth) + 1;
 
-    if (!this.currentPageElement || !this.pageTotal) return;
+    if (this.currentPageElement && this.pageTotalElement) {
+      this.currentPageElement.textContent = this.currentPage;
+      this.pageTotalElement.textContent = this.totalPages;
+    }
 
-    this.currentPageElement.textContent = this.currentPage;
-    this.pageTotal.textContent = this.totalPages;
+    if (this.currentPage != previousPage) {
+      this.dispatchEvent(new CustomEvent('slideChanged', { detail: {
+        currentPage: this.currentPage,
+        currentElement: this.sliderItemsToShow[this.currentPage - 1]
+      }}));
+    }
 
     if (this.enableSliderLooping) return;
 
-    if (this.currentPage === 1) {
+    if (this.isSlideVisible(this.sliderItemsToShow[0])) {
       this.prevButton.setAttribute('disabled', 'disabled');
     } else {
       this.prevButton.removeAttribute('disabled');
     }
 
-    if (this.currentPage === this.totalPages) {
+    if (this.isSlideVisible(this.sliderLastItem)) {
       this.nextButton.setAttribute('disabled', 'disabled');
     } else {
       this.nextButton.removeAttribute('disabled');
```

Comment:
There seem to be a hick-up here. When I try the slider on a feat. collection section I can't get to the end of that slider: 
![](https://screenshot.click/21-12-sdjsp-080x8.png)

Reply:
Fixed! (hopefully)